File: DeQuiDuoi/uocchunglonnhat.cpp
#include <iostream>
using namespace std;

int gcd(int m, int n)
{
  int r;
  if (m < n) return gcd(n,m);
  r = m%m
  if(r == 0)
    return n;
  else 
    return gcd(n, r)
}

int main
{
  int m = 25; 
  int n = 5
  int ketqua = gcd(m, n)
  cout << ketqua;
  return 0
}

File: DeQuiDuoi/uocchunglonnhat.rb
# Tìm ước chung lớn nhất lập trình bằng ruby

def gcd(m, n) 
  @r  #instance varibale
  if(m < n);
    gcd(n,m)
  else
    r = m%n 
    if r==0
      n 
    else
      gcd(n, r)
    end
  end 
end

puts "Ket qua #{gcd(25, 5).to_i}"
puts "Ket qua #{gcd(7, 3).to_i}"

